### DISCUSSION TITLE
Ability to see current states on selected entities in Automation Editor

### DISCUSSION THREAD

**Discussion by cklit** [home-assistant/frontend#11218] [Other feature requests]
## 
I think it could be cool to be able to see the current state for the entities I select for e.g. triggers or conditions directly in the Automation Editor, instead of having to shuffle back and forth between States and Editor :)

**Comment by iantrich** [comment]
How do you propose it be displayed?

**Comment by cklit** [comment]
Hi @iantrich,
I was thinking something like this: (2 examples)

![next to](https://user-images.githubusercontent.com/23294994/73521950-90061900-4407-11ea-86c9-84ffe399c599.png)
![below](https://user-images.githubusercontent.com/23294994/73521951-90061900-4407-11ea-83c0-595bbae01c6b.png)

**Comment by iantrich** [comment]
Or perhaps something like this
![image](https://user-images.githubusercontent.com/1287159/73545955-f7b96580-4401-11ea-816f-7d3917c25722.png)

**Comment by cklit** [comment]
Nice :)

Only downside I can see with this solution is that long entity names will make it difficult to see the state, and that you can’t copy the actual state. 

But that’s also easy to say, when I’m not actually contributing :)

**Comment by iantrich** [comment]
![image](https://user-images.githubusercontent.com/1287159/73570307-c5bff780-4431-11ea-9b69-49db271e9c4d.png)
in the label looks good i think

**Comment by cklit** [comment]
Agree :)
